### PR TITLE
Add Bootstrap to Messages Tab

### LINKED ISSUE #838: Add bootstrap to 'Messages' view components
This is one of many bootstrap tasks. Under the Messages menu, there are various pages. Add bootstrap classes to the elements of these pages. What we want is to make full use of Bootstrap across the entire project.

### PR DESCRIPTION
**Pull request by Jdew192837**
Fixes #838 
 Hello!

Before (Messages):
===
screenshot 5

After (Messages):
===
screenshot 1

Before (Tab):
screenshot 2

After (Tab):
screenshot 3

Thanks,
jdew192837 (nulxis)

### CODE REVIEW

**Comment by Jdew192837** [comment]
@teryhill @aethelwulffe 
PTAL at these implementations I added.
Thanks,
jdew192837 (nulxis)

File: interface/main/dated_reminders/dated_reminders_add.php [hunk 1]
```diff
@@ -23,10 +23,11 @@
...
   $sanitize_all_escapes=true;     
     
-    require_once("../../globals.php"); 
-    require_once("$srcdir/htmlspecialchars.inc.php");  
-    require_once("$srcdir/dated_reminder_functions.php"); 
+  require_once("../../globals.php"); 
+  require_once("$srcdir/htmlspecialchars.inc.php");  
+  require_once("$srcdir/dated_reminder_functions.php"); 
   require_once("$srcdir/formatting.inc.php");
+  require_once("$srcdir/headers.inc.php");
```
Comment (teryhill) [inline interface/main/dated_reminders/dated_reminders_add.php]: Are there any calls that require this? Looks like datepicker is used in this code

File: interface/themes/style_light.css [hunk 2]
```diff
@@ -685,7 +685,7 @@ tr.odd, td.even {
   background: #1050b6;
 }
 
-.css_button, input[type="button"],input[type="submit"], button {
+.css_button, input[type="button"],input[type="submit"], button, a[role="button"] {
```
Comment (teryhill) [inline interface/themes/style_light.css]: Do the other themes need this

File: interface/main/dated_reminders/dated_reminders_add.php (hunk 1 above)
Comment (aethelwulffe) [reply interface/main/dated_reminders/dated_reminders_add.php]: I saw datepicker in a number of things when reviewing today.  Dropping that for HTML5 is a different project, but the call_required_libraries is used immediately below for bootstrap.

File: interface/themes/style_light.css (hunk 2 above)
Comment (aethelwulffe) [reply interface/themes/style_light.css]: If in one, then add to all.

File: interface/main/dated_reminders/dated_reminders_add.php (hunk 1 above)
Comment (Jdew192837) [reply interface/main/dated_reminders/dated_reminders_add.php]: Ah, probably not. Will delete this.

File: interface/themes/style_light.css (hunk 2 above)
Comment (Jdew192837) [reply interface/themes/style_light.css]: Will add now

**Comment by Jdew192837** [comment]
Done @aethelwulffe @teryhill

File: interface/themes/style_pdf.css [hunk 3]
```diff
@@ -605,7 +605,7 @@ font-weight: bold;
     padding: 0px 0px 0px 10px;
 }
 
-.css_button {
+.css_button, a[role="button"] {
```
Comment (aethelwulffe) [inline interface/themes/style_pdf.css]: I don't remember what style_pdf is for at the moment, but it is not actually a full theme...maybe this is appropriate, but I am not sure how to test at the moment.

File: interface/themes/style_tan.css [hunk 4]
```diff
@@ -661,7 +661,7 @@ tr.odd {
 .css_button span, input[type="button"] span, button span {
   display: inline-block;
 }
-.css_button[href*="deleter"], input[type="button"][href*="deleter"], button[href*="deleter"] {
+.css_button[href*="deleter"], input[type="button"][href*="deleter"], button[href*="deleter"], input[type="button"][class="deleter"], button[href*="deleter"], a[class="deleter"] {
```
Comment (aethelwulffe) [inline interface/themes/style_tan.css]: This change was not made in style_purple.css

File: interface/themes/style_purple.css [hunk 5]
```diff
@@ -635,7 +635,7 @@ font-weight: bold;
     padding: 0px 0px 0px 10px;
 }
 
-.css_button {
+.css_button, a[role="button"] {
```
Comment (aethelwulffe) [inline interface/themes/style_purple.css]: need deleter class in this file, right?

**Comment by Jdew192837** [comment]
@aethelwulffe @teryhill Fixed the requested changes, PTAL
Thanks,
jdew192837 (nulxis)

**Comment by Jdew192837** [comment]
@aethelwulffe It still indicates you have changes requested.

**Comment by aethelwulffe** [comment]
Reviews good.  I forgot to actually test the light and tan styles though....

**Comment by aethelwulffe** [comment]
OK, I am having some weird issue with my install at the moment (too much branch tinkering against the data), but I think it works.

Gonna do the merge now.

**Pull request merged**